### PR TITLE
[Tankerkoenig] Solves Issue #2676

### PR DESCRIPTION
**Pull request by JueBag**
Channels of prices for fuel-types that are not reported by a station will be assigned UNDEF (Solving Issue #2676)

Added .toString method for classes OpeningTime and OpeningTimes

Signed-off-by: Jürgen Baginski <opus42@gmx.de>

### CODE REVIEW

**Comment by JueBag** [comment]
I seem to have a problem with Git. 
Quote:

> 23:40:24.911  |  FATAL: Could not checkout master with start point origin/master
> 23:40:24.912  |  hudson.plugins.git.GitException: Command "git checkout -b master origin/master" returned status code 1:
> 23:40:24.912  |  stdout: addons/binding/org.openhab.binding.atlona/.project: needs merge
> 23:40:24.912  |  addons/binding/org.openhab.binding.loxone/.project: needs merge

Why should I have to merge other bindings( atlona and loxone) which I haven't touched?

File: addons/binding/org.openhab.binding.tankerkoenig/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -10,6 +10,7 @@ Import-Package:
  com.google.common.collect,
  com.google.gson,
  org.apache.commons.io,
+ org.apache.commons.lang;version="2.6.0",
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.tankerkoenig/META-INF/MANIFEST.MF]: Is there a reason to use this specific version?

File: addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigDetailResultDeserializer.java [hunk 2]
```diff
@@ -65,4 +70,4 @@ public TankerkoenigDetailResult deserialize(final JsonElement json, final Type t
         }
         return result;
     }
-}
+}
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigDetailResultDeserializer.java]: No new line?

File: addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/OpeningTime.java [hunk 3]
```diff
@@ -47,4 +47,14 @@ public void setEnd(String end) {
         this.end = end;
     }
 
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(this.getText());
+        sb.append(" Open: ");
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/OpeningTime.java]: Just a side note: the StringBuilder.append() method returns itself and could be used for method chaining.

`return new StringBuilder().append(...).append(...)....to String();`

File: addons/binding/org.openhab.binding.tankerkoenig/META-INF/MANIFEST.MF (hunk 1 above)
Comment (JueBag) [reply addons/binding/org.openhab.binding.tankerkoenig/META-INF/MANIFEST.MF]: No! That was created automatically  and I missed to remove it.

File: addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigDetailResultDeserializer.java (hunk 2 above)
Comment (JueBag) [reply addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigDetailResultDeserializer.java]: Coming!

File: addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/OpeningTime.java (hunk 3 above)
Comment (JueBag) [reply addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/OpeningTime.java]: Learned something. Thanks.

**Pull request merged**